src/index.js

```diff
@@ -16,7 +16,6 @@ const deployActions = require('./deploy-actions')
 const undeployActions = require('./undeploy-actions')
 const printActionLogs = require('./print-action-logs')
 const RuntimeAPI = require('./RuntimeAPI')
-const ow = require('openwhisk')
```

Comment:
removing this results in JSDoc warnings for jsdoc for missing type 'ow'
add somewhere at the top of the file:
```
/* global ow */
```